packages/atomic/src/components.d.ts

```diff
@@ -289,6 +289,16 @@ export namespace Components {
     }
     interface AtomicFrequentlyBoughtTogether {
     }
+    interface AtomicHtml {
+        /**
+          * Specify if the content should be sanitized, using `DOMPurify` (https://www.npmjs.com/package/dompurify).
+         */
+        "sanitize": boolean;
+        /**
+          * The string value containg HTML to display;
```

Comment:
```suggestion
          * The string value containing HTML to display;
```

For this and other occurrences of this phrase 🙏